themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties

```diff
@@ -0,0 +1,133 @@
+client_security-admin-console=Console Veligheidsbeheer
+client_admin-cli=Beheer CLI
+client_realm-management=Realm-beheer
+client_broker=Broker
+requiredFields=Verplichte velden
+allFieldsRequired=Alle velden verplicht
+backToApplication=&laquo; Terug naar toepassing
+backTo=Terug naar {0}
+date=Datum
+event=Gebeurtenis
+ip=IP
+client=Client
+clients=Clients
+details=Details
+started=Gestart
+lastAccess=Laatste toegang
+expires=Vervalt
+applications=Toepassingen
+account=Account
+federatedIdentity=Verenigde identiteit
+authenticator=Authenticator
+sessions=Sessies
+log=Log
+application=Toepassing
+availablePermissions=Beschikbare machtigingen
+grantedPermissions=Gegunde rechten
+grantedPersonalInfo=Gegunde Persoonsgegevens
+additionalGrants=Extra gunningen
+action=Actie
+inResource=in
+fullAccess=Volledige toegang
+offlineToken=Offline Token
+revoke=Toekenning terugtrekken
+configureAuthenticators=Ingestelde authenticators
+mobile=Mobiel nummer
+totpStep1=Installeer <a href="https://freeotp.github.io/" target="_blank">FreeOTP</a> of Google Authenticator op uw apparaat. Beide toepassingen zijn beschikbaar in <a href="https://play.google.com">Google Play</a> en de Apple App Store.
+totpStep2=Open de toepassing en scan de QR-code of voer de sleutel in.
+totpStep3=Voer de door de toepassing gegeven eenmalige code in en klik op Opslaan om de configuratie af te ronden.
+missingUsernameMessage=Geef gebruikersnaam alstublieft.
+missingFirstNameMessage=Geef voornaam alstublieft.
+invalidEmailMessage=Ongeldig emailadres.
+missingLastNameMessage=Geef achternaam alstublieft.
+missingEmailMessage=Specificeer email alstublieft.
+missingPasswordMessage=Geef wachtwoord alstublieft.
+notMatchPasswordMessage=Wachtwoorden komen niet overeen.
+missingTotpMessage=Geef authenticatiecode alstublieft.
+invalidPasswordExistingMessage=Ongeldig bestaand wachtwoord.
+invalidPasswordConfirmMessage=Wachtwoordbevestiging komt niet overeen.
+invalidTotpMessage=Ongeldige authenticatiecode.
+emailExistsMessage=Emailadres bestaat reeds.
+readOnlyUserMessage=U kunt uw account niet updaten aangezien het account alleen-lezen is.
+readOnlyPasswordMessage=U kunt uw wachtwoord niet veranderen omdat uw account alleen-lezen is.
+successTotpMessage=Mobiele authenticator geconfigureerd.
+successTotpRemovedMessage=Mobiele authenticator verwijderd.
+successGrantRevokedMessage=Gunning succesvol ingetrokken
+accountUpdatedMessage=Uw account is gewijzigd.
+accountPasswordUpdatedMessage=Uw wachtwoord is veranderd.
+missingIdentityProviderMessage=Geen identity provider aangegeven.
+invalidFederatedIdentityActionMessage=Ongeldige of ontbrekende actie op verenigde identiteit.
+identityProviderNotFoundMessage=Gespecificeerde identity provider niet gevonden.
+federatedIdentityLinkNotActiveMessage=Deze verenigde identiteit is niet langer geldig.
+federatedIdentityRemovingLastProviderMessage=U kunt de laatste verenigde identiteit niet verwijderen aangezien u het wachtwoord niet heeft.
```

Comment:
I understand the meaning now, but that's not in the English translation: `You can''t remove last federated identity as you don''t have password.`